pkg/ccl/streamingccl/streamingest/stream_replication_e2e_test.go

```diff
@@ -306,12 +304,10 @@ func TestTenantStreamingProducerJobTimedOut(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
-	// TODO(casper): now this has the same race issue with
-	// TestPartitionedStreamReplicationClient, please fix them together in the future.
-	skip.UnderRace(t, "disabled under race")
-
 	ctx := context.Background()
-	c, cleanup := createTenantStreamingClusters(ctx, t, defaultTenantStreamingClustersArgs)
+	args := defaultTenantStreamingClustersArgs
+	args.srcClusterSettings[`stream_replication.job_liveness_timeout`] = `'20s'`
```

Comment:
I suspect that might be too low for race tests;
Please verify that your test passes w/ `./dev test --stress --race`